### ISSUE TITLE
[decision-tree] Add "Visualization Tool" icon to "load base map" push button

### ISSUE THREAD

**Issue by luipir** [ConservationInternational/trends.earth#413]
This issue is for the decision-tree branch under development.

The icon to add should be https://github.com/ConservationInternational/trends.earth/blob/master/LDMP/icons/document.svg

**Comment by Samweli** [comment]
@luipir  the icon https://github.com/ConservationInternational/trends.earth/blob/master/LDMP/icons/document.svg is used for the Visualization tool menu action, maybe we need another one for the loading base map action.

**Comment by ricardogsilva** [comment]
@Samweli I agree with @luipir, please use the icon mentioned in the issue description.

The visualization tool menu action is going to be removed at some point in the near future, as we are implementing this functionality in the plugin dock

**Comment by Samweli** [comment]
sure @ricardogsilva, I would prefer another icon that gives a close meaning to "loading a base map", as the proposed icon shows  a document, but I guess I'm the only one here :sweat_smile:.
